docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -113,6 +123,19 @@ The following configuration properties are available:
      - The field of the access token used for the Trino user principal. Defaults to ``sub``. Other commonly used fields include ``sAMAccountName``, ``name``, ``upn``, and ``email``.
    * - ``http-server.authentication.oauth2.groups-field``
      - The field of the access token used for Trino groups. The corresponding claim value must be an array.
+   * - ``http-server.authentication.oauth2.oidc.discovery``
```

Comment:
When this is enabled then some properties mentioned above are not available to user. Maybe we should separate the table in three tables. First with common properties, second with discovery is turned on and third when discover is turned off.

Reply:
All of them are available but some are optional when OIDC discovery is enabled. For example for `http-server.authentication.oauth2.auth-url`
```
 Providing this value while OIDC discovery is enabled overrides the value from OpenID provider metadata document.
```